docs/reference/modules/page.md

```diff
@@ -0,0 +1,379 @@
+
+### `async revertDraftToPublished(req, draft)`
+
+Reverts the given draft page (`draft`) to the most recent publication, clearing any changes. It returns the draft's new value, or `false` if the draft was not modified from the published version or no published version exists yet.
+
+Emits the [`afterRevertDraftToPublished` event](/docs/reference/server-events.md#afterrevertdrafttopublished) before returning, which includes the draft document in its payload, allowing you to alter the returned draft object.
+
+### `async revertPublishedToPrevious(req, published)`
+
+Reverts a published page document (`published`) to the previous published state and returns the updated published state. If this was already done (only one previous state is saved) or there is no previous publication, it throws an `invalid` exception.
+
+Emits the [`afterRevertPublishedToPrevious` event](/docs/reference/server-events.md#afterrevertpublishedtoprevious) before returning, which includes the published document in its payload, allowing you to alter the returned published doc object.
+
+### `normalizeSlug(req)`
+
+Normalizes and replaces `req.slug` to account for unneeded trailing whitespace, trailing slashes other than the root, and double slash based open redirect attempts.
+
+### `isPage(doc),`
+
+Returns `true` if the document object, `doc` is identifiable as a page.
+
+
+### `getBaseUrl(req)`
+
+Returns the effective base URL for the given request (`req`). If Apostrophe's top-level `baseUrl` option is set, or a hostname is defined for the active locale, then that is consulted, otherwise the base URL is the empty string. This makes it easier to build absolute URLs (when `baseUrl` is configured), or to harmlessly prepend the empty string (when it is not configured). The Apostrophe queries used to fetch Apostrophe pages consult this method.
+
+### `inferIdLocaleAndMode(req, _id)`
+
+This method is a wrapper for the `@apostrophecms/i18n` module [method of the same name](/reference/modules/i18n.md##inferidlocaleandmode-req-id)
+
+## Template helpers
+
+Template helpers are methods available for use in template files. Because this module has an alias, you can call these in templates using the alias path. For example, `apos.page.isAncestorOf(doc1, doc2)`.
+
+#### `isAncestorOf(possibleAncestorPage, page)`
+
+Returns a boolean value indicating whether the first argument page object (`possibleAncestorPage`) is an ancestor of the second argument page (`page`) in the page tree.
+
+## Module tasks
+
+### `unpark`
+
+Full command: `node app @apostrophecms/page:unpark /page/slug`
+
+Running this task will unlock a page that was ["parked"](#park) by including its slug as an argument. This allows editors to then change any properties that were not editable previously.
```

Comment:
Note that this is only useful if the page has been removed from the `park` option.